WordPress/Classes/ViewRelated/Media/MediaLibraryPickerDataSource.m

```diff
@@ -615,26 +615,23 @@ - (NSInteger)numberOfAssetsOfType:(WPMediaType)mediaType completionHandler:(WPMe
         [mediaTypes addObject:@(MediaTypeAudio)];
     }
 
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-    NSManagedObjectContext *mainContext = [[ContextManager sharedInstance] newDerivedContext];
-#pragma clang diagnostic pop
-
-    MediaService *mediaService = [[MediaService alloc] initWithManagedObjectContext:mainContext];
-    NSInteger count = [mediaService getMediaLibraryCountForBlog:self.blog
-                                                  forMediaTypes:mediaTypes];
+    NSInteger count = [self.blog mediaLibraryCountForTypes:mediaTypes];
     // If we have a count diferent of zero assume it's correct but sync with the server always in the background
```

Comment:
Unrelated with your changes, but I noticed the typo here, so why not fixing it while we're at it? 😄 

```suggestion
    // If we have a count different of zero, we assume it's correct but we still sync with the server in in the background
```